project-koku/koku
# [COST-4602] - Code changes to use new manifest and report status fields

## Jira Ticket

[COST-4602](https://issues.redhat.com/browse/COST-4602)

## Description

This change will fix up any code use the old field names to the newly renamed fields based on the model changes in https://github.com/project-koku/koku/pull/4881.

## Testing

1. Checkout Branch
2. Restart Koku
3. Make create and load customer data
4. everything should ingest correctly and work as normal

## Notes

THIS WILL REQUIRE QE TEST UPDATES

see: iqe-cost-management-plugin/-/merge_requests/1861

REQUIRES: https://github.com/project-koku/koku/pull/4881
...
